lib/ansible/modules/windows/win_pagefile.ps1

```diff
@@ -0,0 +1,183 @@
+#!powershell
+# This file is part of Ansible
+#
+# Copyright 2017, Liran Nisanov <lirannis@gmail.com>
+#
+# Ansible is free software: you can redistribute it and/or modify
```

Comment:
So, quite recently there was a change to how we add the GPL license to each individual file. It was changed to only add the following one-liner:
```powershell
# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
```